### PR TITLE
fix for issue #251

### PR DESCRIPTION
**Pull request by NezSpencer**
Created a dialog with a 'remember choice' checkbox. when this is checked, the users choice is remembered on subsequent launch of the app, this fixes issue #251

### CODE REVIEW

**Comment by SecUpwN** [comment]
Thanks for your work, @NezSpencer! Would you please fix below Issue so that we can merge this? Not sure if you pulled in the latest changes of our `development` branch before pushing your pull request.

```
:AIMSICD:compileNormalDebugJavaWithJavac/home/travis/build/SecUpwN/Android-IMSI-Catcher-Detector/AIMSIC/src/main/java/com/SecUpwN/AIMSICD/service/AimsicdService.java:266: error: cannot find symbol
        View dialogView=dialogInflater.inflate(R.layout.layout_request_gps,null,false);
                                                       ^
  symbol:   variable layout_request_gps
  location: class layout
/home/travis/build/SecUpwN/Android-IMSI-Catcher-Detector/AIMSID/src/main/java/com/SecUpwN/AIMSICD/service/AimsicdService.java:267: error: cannot find symbol
        CheckBox rememberChoice=(CheckBox)dialogView.findViewById(R.id.check_choice);
                                                                      ^
  symbol:   variable check_choice
  location: class id
/home/travis/build/SecUpwN/Android-IMSI-Catcher-Detector/AIMSID/src/main/java/com/SecUpwN/AIMSICD/service/AimsicdService.java:268: error: cannot find symbol
        Button notNow=(Button)dialogView.findViewById(R.id.not_now_button);
                                                          ^
  symbol:   variable not_now_button
  location: class id
/home/travis/build/SecUpwN/Android-IMSI-Catcher-Detector/AIMSICD/src/main/java/com/SecUpwN/AIMSICD/service/AimsicdService.java:269: error: cannot find symbol
        Button enableGPS=(Button)dialogView.findViewById(R.id.enable_gps_button);
                                                             ^
  symbol:   variable enable_gps_button
  location: class id
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
4 errors
 FAILED
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':AIMSICD:compileNormalDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
```

**Comment by NezSpencer** [comment]
Sorry I noticed it too but my laptop battery was already dead by then. I omitted a layout file. 
I just added it. Try it again

**Comment by NezSpencer** [comment]
and I just pulled in latest changes from main dev branch. I didnt know that before. I am kinda new to all these :)

**Comment by NezSpencer** [comment]
@SecUpwN pls check it now.

**Comment by SecUpwN** [comment]
>@SecUpwN pls check it now.

@NezSpencer, looks better now. Please remove [L27](https://github.com/NezSpencer/Android-IMSI-Catcher-Detector/blob/development/AIMSICD/src/main/java/com/SecUpwN/AIMSICD/service/AimsicdService.java#L27) and I will merge your work.

**Comment by NezSpencer** [comment]
@SecUpwN line 27 removed

**Comment by SecUpwN** [comment]
@larsgrefer, does below error mean that we need to merge #714 in order to make this one work?

```
:AIMSICD:prepareNormalDebugDependenciesModule 'io.freefair.android-util:function:1.1.0' depends on one or more Android Libraries but is a jar
Module 'io.freefair.android-util:logging:1.1.0' depends on one or more Android Libraries but is a jar
 FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':AIMSICD:prepareNormalDebugDependencies'.
> Dependency Error. See console for details.
```

**Comment by larsgrefer** [comment]
The build error has nothing todo with #714

File: AIMSICD/src/main/res/layout/layout_request_gps.xml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (larsgrefer) [inline AIMSICD/src/main/res/layout/layout_request_gps.xml]: using `layout_` as prefix for the filename is redundant, better use `dialog_` as it maches with the existing `activity_` and `fragment_` prefixes

**Comment by NezSpencer** [comment]
@larsgrefer please what could be the cause of the failed build?

**Comment by SecUpwN** [comment]
>@larsgrefer please what could be the cause of the failed build?

@NezSpencer, seems like Travis CI now does not complain anymore. Could you also please do [this](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/pull/744#discussion-diff-49281499)?

**Comment by larsgrefer** [comment]
@NezSpencer The problem was an error with jitpack. I've told them here: https://gitter.im/jitpack/jitpack.io?at=5692c701ee13050b38a2b961 and restarted your travis build afterwards

**Comment by NezSpencer** [comment]
@larsgrefer I have renamed it to dialog_request_gps

**Comment by SecUpwN** [comment]
>I have renamed it to dialog_request_gps

Thanks for your awesome work, hope to see more great fixes from you. Approved!

**Comment by NezSpencer** [comment]
:) thanks

**Pull request merged**